isa_helper.c

```diff
@@ -4,9 +4,7 @@
 uint32_t rotate_right(uint32_t shiftee, uint8_t shifter)
 {
     return (shiftee << (32 - shifter)) | (shiftee >> (shifter));
-}
-
-uint8_t get_bit(uint32_t from, uint8_t bitid)
+} uint8_t get_bit(uint32_t from, uint8_t bitid)
```

Comment:
pls dont play around with whitespace, revert this to original, and i know this is @parthmittal 's fault, but you fix it pls